src/Modules/Backend/assets/installer/translations.xml

```diff
@@ -1025,6 +1025,34 @@
       <translation locale="pt"><![CDATA[utilizadores]]></translation>
       <translation locale="pl"><![CDATA[użytkownicy]]></translation>
     </item>
+    <item type="label" name="2FA">
+      <translation locale="nl"><![CDATA[Two factor authentication]]></translation>
+      <translation locale="en"><![CDATA[Two factor authentication]]></translation>
+    </item>
+    <item type="label" name="EnableTwoFactorAuthentication">
+      <translation locale="nl"><![CDATA[Activeer two factor authentication]]></translation>
+      <translation locale="en"><![CDATA[Enable two factor authentication]]></translation>
+    </item>
+    <item type="label" name="TwoFactorAuthenticationKey">
+      <translation locale="nl"><![CDATA[Autorisatie sleutel]]></translation>
```

Comment:
I suspect this is the number that is shown by the authenticator app? If so: `verificatie code`.

Reply:
No this is the key that is used by the 2fa to generate codes/qr etc